Misc/NEWS.d/next/Library/2024-04-16-21-45-31.gh-issue-117946.ONTFC-.rst

```diff
@@ -0,0 +1 @@
+Adding support for file paths as arguments to pickle.load and pickle.dump. Being able to pass file paths as arguments directly to the load and dump functions help us save some lines of code, especially if you use these functions frequently. You need not explicitly open a file and pass the file object to pickle anymore, you can directly pass the file path instead of the file object. This has been implemented in Python, it still needs to be implemented in the faster, _pickle.c library
```

Comment:
Your blurb should not contain a justification, just a a description of the changes.